docs/src/guides/drupal9/redis.md

```diff
@@ -1,149 +1,172 @@
-If you wish to use Redis for the queue we recommend using a separate persistent Redis instance.
-See the [Redis documentation page](../../add-services/redis.md) for more information.
-
-### Verifying Redis is running
-
-You can verify that Redis is running correctly by connecting to it from an SSH session in your environment.
-After logging in, run
-
-```bash
-echo $PLATFORM_RELATIONSHIPS | base64 --decode | json_pp
-```
-
-to get the list of relationships and find the `host` property for your Redis relationship.
-Then with that value, run
-
-```bash
-redis-cli -h YOUR_REDIS_HOSTNAME info
-```
-
-This should give you a baseline of activity on your Redis installation. There should be very little memory allocated to the Redis cache.
-
-After you push this code, you should run the command and notice that allocated memory will start jumping.
-
-### Clear SQL cache tables
-
-Once you've confirmed that your site is using Redis for caching, you can and should purge any remaining cache data in the MySQL database as it's now just taking up space.
-`TRUNCATE` any table that begins with `cache` *except* for `cache_form`.
-Despite its name `cache_form` isn't part of the cache system proper and thus shouldn't be moved out of SQL.
+Then commit the resulting changes to your `composer.json` 
+and `composer.lock` files.
+
+Alternatively, you can [install the Redis module manually](https://www.drupal.org/project/redis/releases/8.x-1.6), 
+although it is not recommended by Platform.sh or Drupal.
+
+## Configure your Redis service
+
+To configure your Redis service:
+
+1. After [installing the Platform.sh Config Reader library](../../guides/drupal9/deploy/customize.md#install-the-config-reader), 
+   add the following code at the top of your `settings.platformsh.php` file:
+
+   ```php
+    <?php
+
+    $platformsh = new \Platformsh\ConfigReader\Config();
+    if (!$platformsh->inRuntime()) {
+       return;
+    }
+   ```
+
+2. Add the following code at the end of the file:
+
+   ```php {location="settings.platformsh.php"}
+   <?php
+
+   // Enable Redis caching.
+    if ($platformsh->hasRelationship('rediscache') && !InstallerKernel::installationAttempted() && extension_loaded('redis')) {
+      $redis = $platformsh->credentials('rediscache');
+
+      // Set Redis as the default backend for any cache bin not otherwise specified.
+      $settings['cache']['default'] = 'cache.backend.redis';
+      $settings['redis.connection']['host'] = $redis['host'];
+      $settings['redis.connection']['port'] = $redis['port'];
+
+      // Apply changes to the container configuration to make better use of Redis.
+      // This includes using Redis for the lock and flood control systems, as well
+      // as the cache tag checksum. Alternatively, copy the contents of that file
+      // to your project-specific services.yml file, modify as appropriate, and
+      // remove this line.
```

Comment:
:hammer_and_wrench: I think it'd be worth going through a commends and fixing style things. Like the lack of consistency in capitalizing Redis.